samza-core/src/test/java/org/apache/samza/clustermanager/TestContainerRequestState.java

```diff
@@ -193,4 +193,28 @@ public void testContainerAssignment() throws Exception {
 
   }
 
+  @Test
```

Comment:
Could we also add a test for this in TestContainerAllocator and/or TestHostAwareContainerAllocator?

Reply:
I think the scope of the change is quite limited so this test should cover it. The change does not affect the allocator.